### ISSUE TITLE
Add multi-language support

### ISSUE THREAD

**Issue by SirRawlins** [sorry-app/status-bar#65]
We've had lots of requests from customers to make the plugin support multiple languages, picking up that used by the status page itself.

**Comment by SirRawlins** [comment]
The content for the plugin is currently hard-coded into a JS object:

https://github.com/sorry-app/status-bar/blob/master/src/javascripts/status-bar.js#L151-L167

I imagine pulling this out into a more complex object, which contains all the various languages, and then switching out based on the locale of the page returned by the API  would be a decent first approach.

**Comment by SirRawlins** [comment]
Having checked the API documentation, it actually seems that the locale for the page isn't currently an attribute which is returned. So that'll be our first port of call.

Placing this issue on hold until the API presents the information we need.

**Comment by SirRawlins** [comment]
O.k the API now returns the page ISO two-letter locale, which is enough for us to dome a lookup in the plugin and select the right set of content.

**Comment by SirRawlins** [comment]
I've just merged a PR which adds support for languages, it uses the locale from the API, and then does a lookup from a JSON locale file within the plugin which contains all the translations.

**Comment by SirRawlins** [comment]
This was released in `4.5.0` and should be active for all customers once the Cloudfront cache is finished purging.
